requirements.txt

```diff
@@ -6,3 +6,4 @@ boto3==1.10.2
 jsonschema==3.1.1
 falcon==2.0.0
 cheroot==8.2.1
+pyzmq==18.1.1
```

Comment:
We probably gonna need to change grpcio package installation. 
To make it possible for gRPC Server to use SO_REUSE_PORT option you might need to install it with `--no-binary` option like:
`grpcio==1.25.0 --no-binary grpcio`
Unless it's available by default in latest grpcio release. Please check that out.

Reply:
It seems this issue is still open https://github.com/grpc/grpc/issues/18210 and is addresed in also still open PR https://github.com/grpc/grpc/pull/21854
I will update grpcio to 1.27.1 in `requirements.txt` and add workaround `grpcio==1.27.1 --no-binary grpcio --force-reinstall` in dockerfiles until there's a release with fixed `SO_REUSEPORT` flag in grpcio